draft-ietf-taps-interface.md

```diff
@@ -1861,6 +1862,23 @@ lower-bound rate below which the application does not deem
 it will be useful. These are specified in bits per second. 
 The special value `Unlimited` indicates that no bound is specified.
 
+### Group Connection Limit
+
+Name:
+: groupConnLimit
+
+Type:
+: Numeric (with special value `Unlimited`)
+
+Default:
+: Unlimited
+
+This property controls the number of Connections that may arrive from
+a peer as new members of the Connection's group. Similar to SetNewConnectionLimit(),
+this limits the number of ConnectionReceived Events that will occur, but constrained
+to the group of the Connection associated with this property. In case of a multi-streaming
+transport, this takes the role of a limit on the number of allowed streams.
```

Comment:
NiT: /in case of/in the case of/
However: /this take the role of a/
... why so obscure about a role? ... or is there more to be said? ... I don't understand whether there needs to be more said.

Instead, would it be fair just to say:
 /For a multistreaming transport, this limits the numbering of allowed streams./

Reply:
Done, except that I wrote "number" instead of "numbering"  (I don't understand the use of "numbering" here, but I believe that "number" should be ok anyway)